package.json

```diff
@@ -160,6 +160,8 @@
     "@graphql-codegen/typescript-operations": "^3.0.3",
     "@graphql-codegen/typescript-react-apollo": "^3.3.2",
     "@graphql-codegen/typescript-resolvers": "^3.2.0",
+    "@lingui/loader": "^4.4.0",
+    "@lingui/swc-plugin": "^4.0.4",
```

Comment:
just gotta be realll careful with this version. when i last looked into this, it was a very specific combo of next, lingui and swc_core. https://github.com/lingui/swc-plugin#compatibility

we should prob fix this version, wdyt

Reply:
happy to fix